frontend/public/content/module-javascript/hello-javascript.md

```
@@ -1,15 +1,91 @@
-# Hello, JavaScript
+# 🇯‌🇸‌ Hello, JavaScript
 
-Welcome to the JavaScript/TypeScript Execution Sandbox!
+### JavaScript is the programming language of the web. In this lesson, we will learn how to write basic JavaScript functions and see the output directly in the browser, and delve into the topic of Web Workers.
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)